python/common/baseui.py

```diff
@@ -0,0 +1,17 @@
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+
+class BasicFunctions:
+
```

Comment:
This may be an extra return, I cant remember the PIP standard on class spacing.

Reply:
Spacing is acceptable, since PIP suggests one space between class and method declarations.